app/package.json

```
@@ -26,7 +26,7 @@
     "deep-equal": "^1.0.1",
     "dexie": "^2.0.0",
     "double-ended-queue": "^2.1.0-0",
-    "dugite": "1.80.0",
+    "dugite": "1.82.0",
```

we should skip ahead to https://github.com/desktop/dugite/releases/tag/v1.87.0 since that has all the error handling without the bad version of git for windows that required a hotfix last time.
```suggestion
    "dugite": "1.87.0",
```